modules/yieldliftBidAdapter.js

```diff
@@ -0,0 +1,144 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import * as utils from '../src/utils.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const ENDPOINT_URL = 'https://x.yieldlift.com/auction';
+
+const DEFAULT_BID_TTL = 30;
+const DEFAULT_CURRENCY = 'USD';
+const DEFAULT_NET_REVENUE = true;
+
+export const spec = {
+  code: 'yieldlift',
+  aliases: ['yl'],
+  supportedMediaTypes: [BANNER],
+
+  isBidRequestValid: function (bid) {
+    return (!!bid.params.unitId && typeof bid.params.unitId === 'string') ||
+      (!!bid.params.networkId && typeof bid.params.networkId === 'string') ||
+      (!!bid.params.publisherId && typeof bid.params.publisherId === 'string');
+  },
+
+  buildRequests: function (validBidRequests, bidderRequest) {
+    if (!validBidRequests || !bidderRequest) {
+      return;
+    }
+    const publisherId = validBidRequests[0].params.publisherId;
+    const networkId = validBidRequests[0].params.networkId;
+    const impressions = validBidRequests.map(bidRequest => ({
+      id: bidRequest.bidId,
+      banner: {
+        format: bidRequest.sizes.map(sizeArr => ({
```

Comment:
`Adunit.sizes` is [deprecated in PBJS 3](http://prebid.org/blog/pbjs-3), you should access sizes with: `bidRequest.mediaTypes.banner.sizes`

Reply:
fixed in latest commit